base/timing.jl

```diff
@@ -320,16 +342,16 @@ macro timev(msg, ex)
         Experimental.@force_compile
         local stats = gc_num()
         local elapsedtime = time_ns()
-        local compile_elapsedtime = cumulative_compile_time_ns_before()
+        local compile_elapsedtimes = cumulative_compile_time_ns()
```

Comment:
@IanButterworth: doesn't this need to call `cumulative_compile_timing(true)` here as well, to enable compilation timing?

Otherwise, I think this is currently wrong: i think the transformation here isn't enabling compilation time tracking, so you'll probably always see `0` compilation time, unless compilation time tracking is already enabled?


I guess we must not have any tests for this, specifically? :(

Reply:
Ah yes. My bad. Fix here. Haven't run the tests yet, might need work https://github.com/JuliaLang/julia/pull/46100